kumascript/macros/CSSRef.ejs

```diff
@@ -937,6 +937,13 @@ const hasTag = page.hasTag;
 const rtlLocales = ['ar', 'he', 'fa'];
 
 const pageList = await page.subpagesExpand('/en-US/docs/Web/CSS');
+
+// some functions are at depth two
+const levelTwoPages = pageList.reduce((list, page)=>{
+  list.push(...page.subpages);
+  return list;
+}, pageList);
```

Comment:
Let's avoid mutating the `pageList` in the `reduce()` function. How about this?

```suggestion
pageList.push(...pageList.map((page) => page.subpages).flatten());
```

Reply:
`flat()`?